docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM python:3.7
+
+# Use UTF-8 as the default locale
+ENV LANG=C.UTF-8
+
+# Better movement with [Ctrl + →] and [Ctrl + ←]
+ENV INPUTRC=/etc/inputrc
+
+RUN set -ex \
+  && echo '"\e[1;5C": forward-word' >> /etc/inputrc \
+  && echo '"\e[1;5D": backward-word' >> /etc/inputrc
+
+# Set working directory.
+# Mount your project under the same path.
+WORKDIR /usr/src
+
+# Install useful programs
+RUN apt-get update -y && apt-get install -y \
+      zsh \
+      less \
+      git
+
+# Set Zsh as the default shell
+SHELL ["/usr/bin/zsh", "-c"]
+
```

Comment:
I think most people are used to Bash. It's not like we recommend Zsh over Bash in our docs. So this one may be unnecessary? Up to you guys.

Reply:
@jorgeorpinel , using Zsh is not that different from Bash. I could argue that a lot of Bash users are using it because they are not aware of Zsh :sweat_smile: 

I'd prefer using `zsh` over `bash`, since the autocomplete features are better, thus, offering a better experience.

Reply:
Agee about the autocomplete functions. I use Zsh myself 😬 I was just wondering whether a Bash user would find the running container unfamiliar or difficult to use because of differences between Bash and Zsh. But I guess that's not likely. So unless anyone else thinks otherwise, I'm OK with this.